kani-compiler/src/codegen_cprover_gotoc/codegen/typ.rs

```diff
@@ -1130,7 +1130,7 @@ impl<'tcx> GotocCtx<'tcx> {
                     name: field_name,
                     typ: ctx.codegen_ty(field_ty),
                 });
-                offset += field_size;
+                offset = field_offset + field_size;
```

Comment:
Can you please add a comment here? You really only care about the last field, right?

Reply:
What kind of comment do you mean? We also use the offset earlier to figure out how much padding to insert before a field.

Reply:
Gotcha. That's fine, thanks